Odysee/Controllers/MainViewController.swift

```diff
@@ -319,6 +322,99 @@ class MainViewController: UIViewController, AVPlayerViewControllerDelegate {
             // pass
         }
     }
+    
+    func loadLocaleAndCustomBlockedRules() {
+        do {
+            try Lbryio.get(resource: "locale", action: "get", options: [:], completion: { data, error in
+                guard let data = data, error == nil else {
+                    return
+                }
+                if let result = data as? [String: Any] {
+                    self.currentLocale = OdyseeLocale()
+                    self.currentLocale?.continent = result["continent"] as? String
+                    self.currentLocale?.country = result["country"] as? String
+                    self.currentLocale?.isEUMember = true //result["is_eu_member"] as? Bool
```

Comment:
Did you forget to uncomment this out? Should be false by default too (in case API fails?)